applications/application-PwC.md

```diff
@@ -0,0 +1,177 @@
+
+## 1.7. Scope 
+
+PricewaterhouseCoopers (PwC) is a participant in an open ecosystem and the focus of this framework is to describe the process PwC uses for digital exchanges on Indy networks.
+
+## 1.8. Objectives
+
+_This section states the high-level outcomes desired by the trust community through its adoption of the GF._
+
+1. SHOULD specify tangible, achievable results (e.g. SMART criteria and Fit-for-Purpose criteria).
+2. MUST only contain outcomes over which the GF has the authority and mechanisms to achieve within its
+scope.
+3. MUST be consistent with the principles of the GF (below).
+
+
+## 1.9. Principles 
+
+_these are standard principals from ToIP, example to be edited by PricewaterhouseCoopers (PwC)_
+1)	Sustainability
+2)	Stewardship 
+3)	Public good
+4)	Transparency
+5)	Citizen Engagement
+6)	Digital Sovereignty
+7)	Security
+8)	Confidence
+9)	Synchronization
+
+## 1.10. General Requirements
+TBD
+
+## 1.11. Revisions
+Version 1.0
+_TBD - How and when PricewaterhouseCoopers (PwC) would change governance framework_
+
+## 1.12. Extensions
+
+There are no extensions to this Governance Framework.
+
+## 1.13. Schedule of Controlled Documents
+
+TBD
+
+# 2. Controlled Documents
+
+## 2.1. Glossary
+N/A
+
+## 2.2. Risk Assessment
+
+TBD
+
+## 2.3. Trust Assurance and Certification
+
+TBD
+
+## 2.4. Governance Requirements
+
+TBD
+
+## 2.5. Business Requirements
+
+1. PricewaterhouseCoopers (PwC) MUST send an invitation to the other business entity via email to initiate the exchange of information. 
+2. The receiving party MUST accept invitation in order to establish a secure connection.
+
+** Issuance of GHG Credential **
+
+1. Using the established connection, PwC MUST receive a credential request from the business entity. 
+2. Continuing from step 1, the business entity will provide the following attributes in the credential request:
+3. PwC will receive and review the request
+4. Any clarification questions can be asked via the secure messaging function 
+5. Once all data is entered and correct, PwC will offer the credential to the business entity
+6. Business entity can choose to accept or negotiate the credential to request changes
+
+
+**Identity (Authentication)**
+-	VC Storage Location will take place on a digital Business Wallet with a secure connection. 
+-	Verifiable Credential Proof will be requested by ____________ and the proof will be provided by _____________. 
+-	The Verifiable credential will be issued from the _Company_ Digital Wallet
+
+**Delegation**
+-	Refers to a Verifiable Credential for permission to delegate a specific role. 
+-	The VC Storage Location will be held on the digital wallet, with the VC proof being requested by the Company’s Digital wallet. 
+-	VC Proof is provided by the personal digital wallet and the VC will be issued by the Company Digital wallet. 
+
+**Authorization**
```

Comment:
Remove

Reply:
Removed "-	The VC Storage Location will be held on the digital wallet, with the VC proof being requested by the Company’s Digital wallet. 
-	VC Proof is provided by the personal digital wallet and the VC will be issued by the Company Digital wallet. "